src/System.Memory/src/System.Memory.csproj

```diff
@@ -27,12 +24,22 @@
     <Compile Include="System\SpanHelpers.Clear.cs" />
     <Compile Include="System\ThrowHelper.cs" />
   </ItemGroup>
-  <ItemGroup>
+  <ItemGroup Condition="'$(IsPartialFacadeAssembly)' != 'true'">
```

Comment:
Having the exact same public type it multiple different assemblies tends to cause problems. If we wanted to use the CoreLib Unsafe here, it should be fine but it should:

- CoreLib Unsafe should be given different name or namespace, e.g. Internal.Runtime.CompilerServices.
     or 
- CoreLib Unsafe should be filled in to have all methods that S.R.CS.Unsafe has, and S.R.CS.Unsafe should be turned into façade for netcoreapp20